### ISSUE TITLE
[MNT] Inconsistent default for the `n_jobs` argument

### ISSUE THREAD

**Issue by bethrice44** [alan-turing-institute/sktime#3448]
As mentioned [here](https://github.com/alan-turing-institute/sktime/pull/3414), some files use `n_jobs=1` and some use `n_jobs=None` as the default. We should make it consistent.

**Comment by TonyBagnall** [comment]
I agree we should be consistent, personally I would go with n_jobs = 1. Do you have an opinion?

**Comment by fkiraly** [comment]
I think the reasoning @bethrice44 gave to me in favour of `n_jobs=None` makes more sense - in that case, `n_jobs` can be set by the context, whereas in `n_jobs=1` it would always be overridden.

**Comment by bethrice44** [comment]
@fkiraly Yes that's correct but I don't have a strong preference. I imagine most of the time people assume `n_jobs` default will be 1 (i.e. no parallelisation).
